test/integration_test.dart

```diff
@@ -165,11 +165,8 @@ defineTests() {
 
       // https://github.com/dart-lang/linter/issues/246
       test('overrides across libraries', () {
-        dartlint.main([
-          'test/_data/overridden_fields',
-          '-c',
-          'test/_data/overridden_fields/lintconfig.yaml'
-        ]);
+        dartlint.main(
+            ['test/_data/overridden_fields', '--rules', 'overridden_fields']);
```

Comment:
Without this change the integration test failed because the new rule was triggering cc @pq